package.json

```diff
@@ -34,17 +34,17 @@
     "lint:stylelint": "npx stylelint ./packages/**/*.scss --custom-formatter=node_modules/stylelint-formatter-pretty",
     "type-check": "yarn tsc --noEmit",
     "prettier": "npx prettier --write 'packages/**/*.{js,jsx,tsx,scss,html,md,mdx,json}'",
-    "loki": "./loki.sh",
-    "loki:healthcare": "./loki.sh -t healthcare",
-    "loki:medicare": "./loki.sh -t medicare",
+    "loki": "STORYBOOK_DS=core ./loki.sh",
+    "loki:healthcare": "STORYBOOK_DS=healthcare ./loki.sh",
+    "loki:medicare": "STORYBOOK_DS=medicare ./loki.sh",
     "prepare": "husky install",
     "storybook": "STORYBOOK_DS=core start-storybook -p 6006",
     "storybook:healthcare": "STORYBOOK_DS=healthcare start-storybook -p 6006",
```

Comment:
[question] Not directly related to your changes, but do we still need to support the `storybook:healthcare`, `storybook:medicare`, `build-storybook:healthcare` & `build-storybook:medicare` commands?

Reply:
I know we at least need the `storybook:healthcare/medicare` for loki to work appropriately, I don't know that we need to support the build commands though, good question.